reference/examples/LazyCreateAccount2.md

```diff
@@ -0,0 +1,14 @@
+Lazy-create a new account using a public-address via the CryptoTransfer transaction.
+
+Reference: HIP-583
+
+## Example 1
+- Create an Ethereum Account Address / public-address (example: 0xb794f5ea0ba39494ce839613fffba74279579268)
+- Use the `TransferTransaction` and transfer tokens to the Ethereum Account Address / public-address
```

Comment:
What account/key/address is signing the transfer transaction?

Reply:
Same as AccountCreate, this could be any account